Server/Controllers/PostsController.cs

```diff
@@ -0,0 +1,204 @@
+﻿using Server.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Server.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PostsController : ControllerBase
```

Comment:
Should be singular not plural like Post not Posts
Also be specific, it seems its generic Post should be like NewsPost or SocialPost

Reply:
Changed the controller name to SocialPost Controller